src/classes/HH_NameSpec_TEST.cls

```diff
@@ -54,13 +54,13 @@ private with sharing class HH_NameSpec_TEST {
     static testMethod void OneContact() {
         if (strTestOnly != '*' && strTestOnly != 'OneContact') return;
         
-        list<Contact> listCon = new list<Contact>();
-        Contact c1 = new Contact (
-            Salutation = 'Skipper',
-            FirstName = 'John',
-            LastName = 'Doe'
-        );
-        listCon.add(c1);
+        Contact[] listCon = new Contact[] {
```

Comment:
I'd prefer to not have stylistic changes made to code.  it's one thing if we have decided on a team on a new style.  but there are many areas where we have no defined style, and it is currently a matter of choice.  for those undefined styles, you should feel free to use the style you prefer in new code you are writing, but it is recommended that we leave the style alone in existing code.  this reduces the amount of code that the reviewer must review to ensure there are no problems introduced.